app/controllers/v1/service_types_controller.rb

```diff
@@ -191,9 +191,9 @@ def updateAll
           dockerCompose = YAML::load(File.open(directory+'/docker-compose.yml'))
           serviceType = ServiceType.find_by name: mastermindConf["name"], version: mastermindConf["version"]
           if serviceType.nil?
-            ServiceType.create(is_imported: false, project_id: 0, local_path: directory, name: mastermindConf["name"], description: mastermindConf["description"], version: mastermindConf["version"], service_protocol_type: mastermindConf["protocol_type"], ngsi_version: mastermindConf["ngsi_version"], configuration_template: File.read(directory+'/mastermind.yml'), deploy_template: File.read(directory+'/docker-compose.yml'))
+            ServiceType.create(is_imported: false, project_id: 0, local_path: directory, name: mastermindConf["name"], description: mastermindConf["description"], version: mastermindConf["version"], service_protocol_type: mastermindConf["protocol_type"], ngsi_version: mastermindConf["ngsi_version"], configuration_template: File.read(directory+'/mastermind.yml'), deploy_template: File.read(directory+'/docker-compose.yml').gsub(/:-.+}/, "}"))

```

Comment:
Add a comment to state it's a temporary fix so we remember to remove this once the services manager is properly fixed.